snappiershot/snapshot.py

```diff
@@ -165,8 +171,9 @@ def assert_match(self, value: Any, exact: bool = False, update: bool = False) ->
         )
         if not comparison.equal:
             self._snapshot_file.mark_failed(current_index)
-            # TODO: Add human readable diff to AssertionError
-            raise AssertionError
+            diff = self._construct_diff(encoded_value, stored_value, comparison)
+            message = "Snapshot does not match:\n" + diff
```

Comment:
can/should this message include more detail about _which_ snapshot its diffing?

Reply:
We could, but then we'd be double reporting with what pytest already reports